training/new_addresses.xml

```diff
@@ -0,0 +1,12 @@
+<AddressCollection>
```

Comment:
The master training file is `training/labeled.xml`, so these addresses should get appended there.